### PR TITLE
OCPBUGS-19721: Linuxptp daemon overhead ~2x higher with 4.14 operator

### PR DESCRIPTION
**Pull request by aneeshkp**
```
Linux PTP daemon      | 4.13 (OC)  | 4.14(OC)     | 4-14-OC -  FIXED     |     4.14-GM - FIXED
----------------------------------------------------------------------------------------------------------------------------------
Cpu Core          | 9m           |16m              | 9m                        | 11m
Memory            | 20Mi         | 20Mi            | 15Mi                      | 34Mi
```

### CODE REVIEW

File: pkg/event/event.go [hunk 1]
```diff
@@ -456,28 +456,35 @@ connect:
...
+				StateRegisterer.monitor()
+			}
+		}
+	}()
 
 	glog.Info("starting state monitoring...")
 	for {
 		select {
-		case event := <-e.processChannel:
+		case event := <-e.processChannel: //for NON GM thi will in sleep forever
```
Comment (josephdrichard) [inline pkg/event/event.go:480]: fix comment

Comment (aneeshkp) [reply pkg/event/event.go]: Done

**Review by josephdrichard — APPROVED** [review]
/lgtm
/approve

**Pull request merged**